code/__HELPERS/game.dm

```diff
@@ -199,11 +199,9 @@
 /proc/get_hearers_in_view(R, atom/source)
 	// Returns a list of hearers in view(R) from source (ignoring luminosity). Used in saycode.
 	var/turf/T = get_turf(source)
-	. = list()

```

Comment:
You're changing it to return null instead of empty list by default, nothing cares about this ?

Reply:
oh
haha wait
fuck